app/javascript/movie_search.js

```diff
@@ -0,0 +1,41 @@
+$(function () {
```

Comment:
`turbolinks` の読み込みを待つようにしましょう。

```suggestion
$(document).on('turbolinks:load', function() {
```